storage-logzio/src/test/java/LogzioSpanConsumerTest.java

```diff
@@ -122,32 +126,32 @@ public void interruptSenderCloseTest() {
       closingThread.interrupt();
    }
 
+   /** This shows that calls are in fact async, and cancelable */
```

Comment:
I wasn't sure what this test was aiming to prove, but it was flakey and I took a guess that this was about cancellation.

Reply:
@adriancole this test checks an interrupt while sending traces. when this happens, the extension should wait for the sender to complete the send operation and only then to close itself. this is why expect to see the whole 100

Reply:
cool I reverted my change adding a comment